src/RoslynInsertionTool/RoslynInsertionTool.Commandline/RoslynInsertionToolCommandline.cs

```diff
@@ -6,12 +6,18 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Web.Configuration;
+
 using Microsoft.Azure.KeyVault;
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
+
 using Mono.Options;
+
```

Comment:
This looks...curious?

Reply:
This is the new add-using feature that groups common namespaces and separates others out.